pkger/parser.go

```diff
@@ -138,8 +148,14 @@ func parseSource(r io.Reader, opts ...ValidateOptFn) (*Pkg, error) {
 	switch {
 	case strings.Contains(contentType, "json"):
 		return parseJSON(bytes.NewReader(b), opts...)
-	default:
+	case strings.Contains(contentType, "jsonnet"):
```

Comment:
strings.Contains(contentType, "json") will always be true, even if the "content-type" is "jsonnet"

Reply:
ohhhh great catch 🥇, will fix that up 👍

Reply:
@kelwang push fixed